themes/bootstrap5/templates/Helpers/pagination.phtml

```diff
@@ -1,24 +1,46 @@
 <?php if ($this->pageCount): ?>
   <?php
+    $results = $this->params['results'] ?? null;
     $showFirstLast = $this->pageCount > 3;
+    $navAttrs = $this->htmlAttributes(['aria-label' => $this->translate('pagination_label')]);
+    if ($results !== null) {
+      $showFirstLast = $showFirstLast && $results->getOptions()->firstLastNavigationSupported();
+      if ($results->getOptions()->loadResultsWithJsEnabled()) {
+        $navAttrs->add('class', 'js-pagination');
+      }
+    }
+    if ($this->top) {
+      $navAttrs->add('class', 'js-pagination__top');
```

Comment:
This should only be added if `js-pagination` is added as well, i.e. loadResultsWithJsEnabled is enabled.

Reply:
Fixed